flax/nnx/helpers.py

```diff
@@ -99,7 +102,17 @@ def __call__(self, *args, rngs: tp.Optional[Rngs] = None, **kwargs) -> tp.Any:
 
     return output
 
-
+  @staticmethod
+  def identity(*args, rngs: tp.Optional[Rngs] = None, **kwargs) -> tp.Any:
```

Comment:
@SobhanMP can we just do this logic under the if statement above instead of creating a new method?